casper/src/main/scala/coop/rchain/casper/Validate.scala

```diff
@@ -418,6 +422,43 @@ object Validate {
       } yield Left(InvalidDeployCount)
     }
 
+  // TODO: Double check logic with validator rotation
+  def lastSequenceNumber[F[_]: Monad: Log: BlockStore](
```

Comment:
The name of this fucntion is misleading. Its name implies a computed lastSEquenceNumber while it actually does a validation

Reply:
Except it is in the Validate class. What do you suggest?

Reply:
Actually you are right. I didn't get that the class name is the verb here. Never mind this comment